membrane_curvature/core.py

```diff
@@ -5,60 +5,42 @@
 Handles the primary functions
 """
 
-from .lib.mods import dict2pickle, core_fast, curvature
-import sys
-import os
+from numpy.core.fromnumeric import mean
+from .lib.mods import curvature, core_fast_leaflet, gaussian_curvature, mean_curvature
 import time
-import mdtraj as md
 import MDAnalysis as mda
 import math
 from pathlib import Path
 __author__ = "Estefania Barreto-Ojeda"
 version = 0.1
 
-sys.path.append('lib/')
 
 def main():
 
     start_time = time.time()
 
-    # 3. Define leaflets and lipid type
+    # 1. Define leaflets and lipid type # Gone in next steps
     leaflets = ['lower', 'upper']
 
-    #lipid_types = [ ]
-    lipid_types = ['POPC', 'POPE']
-
-    # 4. head indexes
-    head_index = [iil, jjl, jju]
-
-    # 5. Assign topology
-    topology = md.load(grofile).topology
-
-    # 6. Populate universe with coordinates and trajectory
+    # 2. Populate universe with coordinates and trajectory
     u = mda.Universe(grofile, trjfile)
 
-    # 6.1 Set grid: Extract box dimension from MD sim,
+    # 3 Set grid: Extract box dimension from MD sim,
     # set grid max width, set number of unit cells
     box_size = u.dimensions[0]
```

Comment:
This gives you Lx only. Do you assume square X-Y ?

Reply:
Maybe at least
```python
box_size = max(u.dimensions[0], u.dimensions[1])
```

If you assume orthorhombic boxes then consider just failing here if you encounter a triclinic box.

Reply:
Yes, this is one of the limitations and one of the next issues to solve. Issue #35 added.